### PR TITLE
mc10: reduce video window to hide vertical retrace lines

### PR DESCRIPTION
**Pull request by tlindner**

### CODE REVIEW

File: src/mame/drivers/mc10.cpp [hunk 1]
```diff
@@ -513,7 +513,7 @@ void mc10_state::mc10_video(machine_config &config)
 	RAM(config, m_ram).set_default_size("4K").set_extra_options("8K,20K,32K");
 
 	/* video hardware */
-	SCREEN(config, "screen", SCREEN_TYPE_RASTER).set_raw(9.828_MHz_XTAL / 2, 320, 0, 320, 243, 0, 243);
+	SCREEN(config, "screen", SCREEN_TYPE_RASTER).set_raw(9.828_MHz_XTAL / 2, 320, 0, 320, 243, 0, 240);
```
Comment (angelosa) [inline src/mame/drivers/mc10.cpp:516]: Same screen h/v size as visibile area!? Can't possibly be right, and I'd be surprised if it won't cause all kinds of side effects.
Do we have screen timing parameters for this?

**Review by angelosa — CHANGES_REQUESTED** [review]

File: src/mame/drivers/mc10.cpp (hunk 1 above)
Comment (startaq) [reply src/mame/drivers/mc10.cpp:516]: Should be common between all MC6847 machines (PAL/NTSC versions).

Comment (tlindner) [reply src/mame/drivers/mc10.cpp:516]: The mc10 was the first (and only) device changed to this interface. I see now how I have no horizontal blanking time. If anyone wants to provide help I could use it.

Comment (tlindner) [reply src/mame/drivers/mc10.cpp]: I think I have this much better.

**Pull request merged**